pkg/plugin/plugin.go

```diff
@@ -41,10 +41,14 @@ type Plugin interface {
 	// We don't use onion model like Gin/Caddy because we don't serve the whole request lifecycle
 	// inside the runner. The plugin is only a filter running at one stage.
 	Filter(conf interface{}, w http.ResponseWriter, r pkgHTTP.Request)
+
+	// RespFilter is the method to handle response.
+	// This filter is currently only pre-defined and has not been implemented.
+	RespFilter(conf interface{}, w pkgHTTP.Response)
```

Comment:
As we already introduce break change, what about renaming Filter to RequestFilter?